demos/face_blur/python/Makefile

```diff
@@ -24,9 +24,10 @@ setup_repository:
 	curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/face-detection-retail-0004/FP32/face-detection-retail-0004.xml -o workspace/face-detection-retail-0004/1/face-detection-retail-0004.xml
 	curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/face-detection-retail-0004/FP32/face-detection-retail-0004.bin	-o workspace/face-detection-retail-0004/1/face-detection-retail-0004.bin
 # Build face_blur custom node
-	cd ../../../src/custom_nodes/face_blur && \
-	make && \
-	mv lib ../../../demos/face_blur/python/workspace/lib
+	cd ../../../src/custom_nodes && \
+	make BASE_OS=ubuntu NODES=face_blur && \
```

Comment:
We have RedHat in src/custom_nodes/face_blur/README.md

"You can also select base OS between RH 8.5 (redhat) and Ubuntu 20.04 (ubuntu) by setting `BASE_OS` environment variable."

Shouldn't we correct README?

Reply:
In ` src/custom_nodes/face_blur/README.md` we document that it is possible to build for ubuntu and redhat version of OVMS.

In demo we show one specific use case, for ubuntu.